service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/FurtherEvidenceUploadedEventHandler.java

```diff
@@ -200,10 +202,18 @@ public void sendHearingDocumentsUploadedNotification(final FurtherEvidenceUpload
             recipients.addAll(furtherEvidenceNotificationService.getLocalAuthoritiesRecipients(caseData));
 
             if (isNotEmpty(recipients)) {
+                List<Element<HearingBooking>> hearingBookings = caseData.getHearingDetails().stream()
+                    .filter(element -> element.getValue().toLabel().equals(newHearingDocuments.get(0).getHearing()))
+                    .collect(Collectors.toList());
+
+                Optional<HearingBooking> hearingBookingToSend = hearingBookings.isEmpty()
+                    ? Optional.empty()
+                    : Optional.of(hearingBookings.get(0).getValue());
```

Comment:
I think we might be able to simplify this by replacing `.collect` with `.findFirst` (returns an optional of the first element in the stream if it exists, or Optional.empty if not)